packages/client/src/app/game/views/game-map/me/me.component.ts

```diff
@@ -17,16 +17,21 @@ import { CharacterData } from '../../../../types';
 })
 export class MeComponent implements OnInit, OnDestroy {
 
+  private meModelDrawSubscription: Subscription;
+
   @ViewChild('cross') crossElement: ElementRef;
   @ViewChild('gunShotSound') gunShotSound: ElementRef;
   @ViewChild('muzzleFlash') muzzleFlash: ElementRef;
+  @ViewChild('meModel') meModel: BasicDesc;
 
   showWeapon$: Observable<boolean>;
   showCross$: Observable<boolean>;
   clickSub$: Subscription;
   ViewState = ViewState;
   buildingNearby = false;
   insideBuilding = false;
+  transparentColor = new Cesium.Color(0, 0, 0, 0.01);
```

Comment:
y did you use transpernt color to remove the entity? what wrong with show?

Reply:
cant fly to a hidden entity - no bounding sphere